shared/common-adapters/section-list.desktop.tsx

```diff
@@ -236,7 +236,7 @@ class SectionList<T extends Section<any, any>> extends React.Component<Props<T>,
       <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.container}>
         {this.props.disableAbsoluteStickyHeader && stickyHeader}
         <Kb.ScrollView
-          style={Styles.collapseStyles([styles.scroll, this.props.contentContainerStyle])}
+          style={Styles.collapseStyles([styles.scroll, this.props.style, this.props.contentContainerStyle])}
```

Comment:
Desktop scrollview has `contentContainerStyle`, can we use that? and change the comment in `d.ts` about it being mobile only.